pipelines/push-disk-images-to-cdn/push-disk-images-to-cdn.yaml

```diff
@@ -4,7 +4,7 @@ kind: Pipeline
 metadata:
   name: push-disk-images-to-cdn
   labels:
-    app.kubernetes.io/version: "1.2.0"
+    app.kubernetes.io/version: "1.2.1"
```

Comment:
If I am not mistaken, for new parameters you should update the minor  - or 1.3.0 instead.

Reply:
Fixed